docroot/modules/custom/mass_flagging/mass_flagging.module

```diff
@@ -142,7 +142,7 @@ function mass_flagging_node_update_insert_send_notifications(Node $node) {
               '@node_edit_time' => $node_edit_time,
               ':diff_url' => $diff_url,
               '@author_message' => $author_message ? "\n\nRevision log message:\n\n" . $author_message : '',
-              ':help_url' => Url::fromRoute('help.page', ['name' => 'mass_flagging'])->setAbsolute()->toString(),
+//              ':help_url' => Url::fromRoute('help.page', ['name' => 'mass_flagging'])->setAbsolute()->toString(),
```

Comment:
We should delete this line - it's causing test failures for code style.

Reply:
done

Reply:
Resolve this as the requested change was done.